README.md

```diff
@@ -6,6 +6,7 @@
   * [OrionHDFSSink persistence](#section3.1)
```

Comment:
An entry in the CHANGES_NEXT_RELEASE file corresponding to this PR should be needed.